### PR TITLE
SP2-630 add unit tests for RemoveGoalController

### PR DESCRIPTION
**Pull request by pk2453**

### CODE REVIEW

File: server/routes/removeGoal/RemoveGoalController.test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  })
+
+  describe('post', () => {
+    it('should return to plan-summary if cancel removal', async () => {
+      req = { body: { type: 'some-type', action: 'cancelRemove' } } as Request
+      await controller.post(req as Request, res as Response, next)
+      expect(res.redirect).toHaveBeenCalledWith(`${URLs.PLAN_SUMMARY}?type=some-type`)
+    })
+
+    it('should return to plan-summary with removing goal if cancel removal', async () => {
```
Comment (pipwilson) [inline server/routes/removeGoal/RemoveGoalController.test.ts]: isn't this the same test as above but with an additional assert?

Comment (pk2453) [reply server/routes/removeGoal/RemoveGoalController.test.ts]: Yeah didnt realized that, removed one.

Comment (pipwilson) [reply server/routes/removeGoal/RemoveGoalController.test.ts]: should the title in the `it` line be "with*out* removing goal" ?

Comment (pk2453) [reply server/routes/removeGoal/RemoveGoalController.test.ts]: > should the title in the `it` line be "with_out_ removing goal" ?
yes its updated now

**Review by pipwilson — APPROVED** [review]
LGTM

**Pull request merged**